README.md

```diff
@@ -201,7 +186,7 @@ export const User = {
   attributes: {
     name: "STRING",
   },
-  hasMany: [{ target: "Todo", options: { as: "todos" } }], // 👀
```

Comment:
the 👀 is there to draw attention to line changes, we were originally using them when we were adding new lines to previous snippets, but in this case i think it's ok (i do concede it might be a bit confusing) because the section immediately after talks about relationships and `hasMany` etc so _hopefully_ some people will think of this as some special/magic line they need to bring their attention to

Reply:
Just to clarify: would you like me to keep the 👀 or remove them? If we keep them, I'll add another `> Note` or maybe even a key of some sort so that users know exactly what they signify

Reply:
lets keep them! note sounds good